### ISSUE TITLE
NI@0.10 breaks at first breakpoint, but still runs program on Node@0.12

### ISSUE THREAD

**Issue by fiznool** [node-inspector/node-inspector#630]
```
➜  node -v
v0.12.2

➜  node-debug -v
Node Inspector v0.10.0

➜  cat index.js
var msg = 'this should be logged after execution is resumed';
console.log(msg);

➜  node-debug index.js --cli
Node Inspector is now available from http://127.0.0.1:8080/debug?ws=127.0.0.1:8080&port=5858
Debugging `index.js`

Debugger listening on port 5858
```

I then open `http://localhost:8080/debug?ws=127.0.0.1:8080&port=5858` in my browser window. When node-inspector opens, I see this:

![screenshot from 2015-05-01 18 14 12](https://cloud.githubusercontent.com/assets/820863/7433924/1687c3ee-f02e-11e4-8ada-46158c1c335a.png)

Looks good - however, the terminal window has logged out the message:

```
this should be logged after execution is resumed
```

which should only be displayed once I press F8 to resume execution.

Ubuntu 14.10, node-inspector 0.10.0, node 0.12.2.

Any idea what might be causing this?

**Comment by clintwood** [comment]
Same here on Win 8.1
Node Inspector v0.10.0
io.js v2.0.0-nightly20150504f34b105ccd :)

**Comment by 3y3** [comment]
I can't reproduce this behavior on Win8.1 with 0.12.2
@bajtos , can you confirm this as bug on your Mac?

**Comment by DaHoopster** [comment]
I can confirm the same issue on Mac OSX Yosemite, 10.10.2 with Node.js v0.12.0 and node-inspector 0.10.0

**Comment by 3y3** [comment]
Maybe https://github.com/node-inspector/node-inspector/pull/637

**Comment by 3y3** [comment]
Please try to reproduce this on node-inspector 0.10.1

**Comment by fiznool** [comment]
Fixed as of v0.11.0. :hamster:
